examples/java/spring-boot/build.gradle

```
@@ -16,7 +16,7 @@ dependencies {
 }
 
 jsonSchema2Pojo {
-  source = files("../../../schemas/v1alpha")
+  source = files(fileTree(dir:'../../../schemas/v1alpha', include:['*.json']))
```

If Spotless allows it:
```suggestion
  source = files(fileTree(dir: '../../../schemas/v1alpha', include: ['*.json']))
```